### PR TITLE
Actually fix Stale Bumper

### PR DESCRIPTION
**Pull request by OmarShehata**
Last time I "fixed" https://github.com/AnalyticalGraphicsInc/cesium-concierge/issues/116 it was because of line endings. I confirmed that the cron script was running, called it a day, and moved on with my life. 

The cron script _is_ running, but since the system initiates the cron script, not the root user, it has a different environment, so all the AWS environment settings aren't available to the cron script. To remedy this without having to manually specify the secret key somewhere, this PR makes it dump the environment on deploy into a file, which the cron script can then grab when running. 

I believe this is the easiest/most automated way.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata! 
* :heavy_check_mark: Signed CLA found. 
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium-concierge/blob/fix-cron/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

---

:sparkles: :sparkles: :sparkles:__I'm the Cesium Concierge and I'm here to test things.__ :sparkles: :sparkles: :sparkles:

**Comment by OmarShehata** [comment]
Just tested this, but the Docker command to dump the env runs too early, and the env doesn't include the variables from AWS. Looking for a better way.

**Comment by OmarShehata** [comment]
Ok, I'm pretty confident this should work @ggetz . It writes out the env from the application to ensure it's actually the same settings. I'm also fairly confident this is secure because if someone has access to the disk, then they already have access to the environment variables on that system.

**Comment by ggetz** [comment]
@OmarShehata Should we do any cleanup of the environment file at the end? I'm still not thrilled at saving all the environment variables to the system and leaving them there...

File: lib/Settings.js [hunk 1]
```diff
@@ -13,6 +14,20 @@ var Settings = {};
 
 var repositories = {};
 
+function dumpEnvironment(obj, parentName) {
+    var envString = '';
+    Object.keys(obj).forEach(function(name) {
+        var fullName = parentName === '' ? name : parentName + '__' + name;
```
Comment (ggetz) [inline lib/Settings.js]: Is there a way to do this with `nconf` directly? Like https://www.npmjs.com/package/nconf#file?

**Comment by OmarShehata** [comment]
It must remain there every time the cron script runs, since the cron script always runs with a barebones environment. 

I have been digging and I don't see any potential issue with this scheme. I think the more traditional approach would be to store them in `~/.bashrc` and have the cron script get its environment variables from there, but that's the same thing (except the file lives somewhere else). Security-wise I can't see any situation where someone would have access to the filesystem and not to the environment variables.

File: lib/Settings.js (hunk 1 above)
Comment (OmarShehata) [reply lib/Settings.js]: Yeah I was looking at that. I believe that outputs it as a JSON whereas we would need to convert it back to a flat list joined with `__` anyway.

**Comment by ggetz** [comment]
OK, let's merge and give this a try.

**Pull request merged**